src/main/java/core/basesyntax/CountSupplyAndBuy.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class CountSupplyAndBuy {
+    private final String replaceChar = "";
+    private final int initialAmount = 0;
```

Comment:
in this case `replaceChar` and `initialAmount` may be redundant
we can see their meaning from the context